travis.sh

```diff
@@ -79,3 +81,14 @@ if [ -n "${XVFBPID:-}" ]; then
     kill $XVFBPID
     wait
 fi
+
+if [ -n "${COVERALLS_SERVICE_NAME}" ]; then
+echo "travis build dir: ${TRAVIS_BUILD_DIR}"
+    lcov --directory ./build --capture  --output-file coverage.info
+    lcov --remove coverage.info "/usr/*" --output-file coverage.info
+    lcov --remove coverage.info "${TRAVIS_BUILD_DIR}/build/gmock/*" --output-file coverage.info
```

Comment:
I think you can exclude `build/*`. That would also get rid of the generated `moc_CalculatorWidget.cpp` and `cpp_steps.cpp` neither of which we care about for coverage.